src/bootstrap/bootstrap.py

```diff
@@ -888,7 +888,11 @@ def build_bootstrap(self, color, verbose_count):
         if target_linker is not None:
             env["RUSTFLAGS"] += " -C linker=" + target_linker
         env["RUSTFLAGS"] += " -Wrust_2018_idioms -Wunused_lifetimes"
-        if self.get_toml("deny-warnings", "rust") != "false":
+        if warnings == "default":
```

Comment:
I gave you the wrong suggestion code, sorry, this is unset by default
```suggestion
        if warnings is None or warnings == "default":
```
(or add `default="default"` in the argparse code below)

Reply:
Uhh, argparse is a piece of crap :D I always use `click` or `typer`, so I forgot about this.